go/vt/wrangler/vexec.go

```diff
@@ -385,6 +396,8 @@ type ReplicationStatus struct {
 	TimeUpdated int64
 	// Message represents the message column from the _vt.vreplication table.
 	Message string
+	// Tags contain the tags specified for this stream
+	Tags string
```

Comment:
We'll also need to update the protos for this field and the corresponding impl in `vtctl/workflow/server.go`

Reply:
Added and verified that the tags are shown in the JSON response for the workflow streams in vtadmin.